application/account-management/WebApp/src/translations/locale/da-DK.po

```diff
@@ -0,0 +1,17 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-12-13 18:10+0100\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: da-DK\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+msgid "Overview"
+msgstr "Oversigt"
```

Comment:
Are `msgid` and `msgstr` our own. Can we avoid the abbrivations?

Reply:
it's part of the format https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html